modules/getting-started-openshift-common-terms.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-overview.adoc
+
+:_content-type: REFERENCE
+[id="getting-started-openshift-common-terms_{context}"]
+= Common Terms
+
+The following are common terms that are used within the Getting Started. These terms help you orient yourself with the content and other parts of the documentation.
```

Comment:
I would specify that these are OCP/Kubernetes terms. Also "within the Getting Started" doesn't sound right. I'd suggest rewording this intro a bit. Maybe something like "This glossary defines common Kubernetes and {product-title} terms."